docs/Installation.md

```diff
@@ -68,45 +68,48 @@ sudo systemctl reload httpd
 
 ### 2. Debian
 
-#### Install apache2 and disable default site
+#### Install Apache
 
 ```sh
 sudo apt-get update && sudo apt-get upgrade -y 
-sudo apt-get install apache2 unzip
+sudo apt-get install -y apache2 unzip
+```
+
+#### Basic Apache configuration
+
+```sh
 sudo a2enmod proxy
 sudo a2enmod proxy_http
 sudo a2enmod rewrite
 sudo a2dissite 000-default
+sudo systemctl enable apache2
 sudo systemctl restart apache2
 ```
 
 #### Install Zonemaster Web GUI
 
 ```sh
 wget https://github.com/zonemaster/zonemaster-gui/releases/download/v3.2.0/zonemaster_web_gui.zip -O zonemaster_web_gui.zip
-sudo mkdir -p  /var/www/html/zonemaster-web-gui
-sudo mkdir -p /var/log/zonemaster
 sudo unzip -d /var/www/html/zonemaster-web-gui zonemaster_web_gui.zip
-rm zonemaster_web_gui.zip
+sudo install -vd /var/www/html/zonemaster-web-gui
```

Comment:
This line is not needed because the `unzip` command already creates the folder.

Reply:
Good point!